spinalcordtoolbox/scripts/sct_register_to_template.py

```diff
@@ -135,6 +135,15 @@ def get_parser():
         action="help",
         help="Show this help message and exit."
     )
+    optional.add_argument(
+        '-s-template-id',
+        metavar=Metavar.int,
+        type=int,
+        help="Segmentation from the template to associate with the flag '-s'. Typically, the spinal cord segmentation "
+             "is used, but if it is available, the white matter mask would produced better registration results. The ID"
```

Comment:
Is this "would" or "could"?

Reply:
good catch-- could is more accurate-- can you pls update the code-- thx